File: CS50x/Week6/pset6/sentimental-mario-less/mario.py
# Author: Eskabore
# Purpose: buiding a left-aligned pyramid
# Language: Python

from cs50 import get_int


def main():
    # Prompt user for input & built Pyramid
    height = get_int("Enter number between 1 to 8: ")
    while height < 1 or height > 8:
        height = get_int("Type an integer between 1 to 8: ")

    # Draw a right-aligned pyramid based on input value
    for i in range(1, height+1):
        # Inner loop that displays the 'columns' or 'spaces'
        for j in range(height, i, -1):
            print(" ", end="")
        for k in range(1, i+1):
            print("#", end="")
        print()


if __name__ == "__main__":
    main()

File: CS50x/Week1/pset1/mario/mario-less/mario.c
/*
* Author: Eskabore
* Purpose: building a right-aligned pyramid
* Language C
*/
#include <cs50.h>
#include <stdio.h>

/* Prompt user for input & built Pyramid */
int main(void)
{
    int height;

    /* Prompt user input until value is > 0 & < 9 */
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);


    /* Draw a right-aligned pyramid based on input value */
    for (int i = 1; i <= height; i++)
    {
        /* Inner loop that displays the 'columns' or 'spaces' */
        for (int j = height; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}